spec/unit/speak_slack_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../app/speak_slack'
+require_relative '../../constants'
+
+module FoodTruck
+  RSpec.describe SpeakSlack do
+    describe 'SpeakSlack says' do
```

Comment:
Here too, I'm not sure what purpose this nested example group serves. Notice the documentation output:

```
FoodTruck::SpeakSlack
  SpeakSlack says
    when passed an empty array
      food trucks cannot be found
    when passed an array with food trucks
      lists food trucks in a slack format
      lists uniq food trucks
```

I'm not sure what "SpeakSlack says" communicates that the outer bit ("FoodTruck::SpeakSlack") did not already communicate.